### PR TITLE
Update with PGD fitting script + some jacobian scripts

### PR DESCRIPTION
**Pull request by EiffL**
This PR brings a few updates such as dedicated scripts for computing the jacobians and PGD parameters.

Still a draft for now, because it is a little bit of a mess there are too many random scripts

### CODE REVIEW

**Comment by EiffL** [comment]
We need to normalize somewhere the way we are making maps... it's very dangerous to have the same code in multiple scripts.

File: DifferentiableHOS/statistics/l1norm.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  """ Computes normalizing constant for starlet, for given image.
+  """
+  _, nx, ny = image.get_shape()
+  knorm = tf.ones((1, 1, 1, 1), dtype=tf.float32)
+  knorm = tf.image.resize_with_crop_or_pad(knorm, nx, ny)
+  wt = starlet2d(knorm[..., 0], nscales=nscales)
+  return [tf.math.sqrt(tf.reduce_sum(c**2)) for c in wt]
+
+
+def l1norm(image, nscales=3, value_range=[-1., 1.], nbins=16, name='l1norm'):
```
Comment (dlanzieri) [inline DifferentiableHOS/statistics/l1norm.py:20]: is this one the differentiable implementation of the  l1norm?

**Review by dlanzieri — CHANGES_REQUESTED** [review]

File: DifferentiableHOS/statistics/l1norm.py (hunk 1 above)
Comment (EiffL) [reply DifferentiableHOS/statistics/l1norm.py:20]: hum no :-( sorry I forgot to push it last night,

**Review by dlanzieri — CHANGES_REQUESTED** [review]
I suggest some modifs, but It looks ok for me

**Comment by EiffL** [comment]
Ok, thanks Denise!

**Review by dlanzieri — APPROVED** [review]
